### DISCUSSION TITLE
Comment utiliser la notion de groupe

### DISCUSSION THREAD

**Discussion by SteBoun** [JeromeDevome/GRR#336] [General]
Bonjour,

Je voudrais utiliser la notion de groupe, car j'ai remarqué que quand j'exporte mes utilisateurs LDAP sur GRR, les utilisateurs n'ont pas leur groupe AD.
J'ai vu le champ "groupe" dans les paramètres GRR, mais je ne sais pas comment les utiliser les groupes dans GRR parce que je dois gérer les accès à de grand groupe d'utilisateur.

**Comment by ynaessens** [comment]
Bonjour,
la notion de groupe dans GRR est encore à l'état d'ébauche.
Si vous pouvez contribuer à la faire avancer, vous êtes bienvenu.
Cordialement,
YN
